src/base.ts

```diff
@@ -18,12 +18,22 @@ const userAgent = `Airtable.js/${packageVersion}`;
 type BaseRequestOptions = {
     method?: string;
     path?: string;
-    qs?: any;
-    headers?: any;
-    body?: any;
+    qs?: Record<string, any>;
```

Comment:
@SomeHats in [Airtable/Base: Pass custom headers to requests made through the client](https://github.com/Airtable/airtable.js/pull/213/commits/5bbccb3856ade7db8e8ef18ec8db3d46616bc9ee), I've introduced the `ObjectMap` we discussed in Slack. Once this PR lands and then that PR lands, we can come back use `ObjectMap` here as well.

Reply:
sounds good to me!